test/index.js

```diff
@@ -257,13 +257,24 @@ test('NSpell()', function (t) {
 
     st.deepEqual(
       us.suggest('propper'),
-      ['dropper', 'proper', 'cropper', 'popper', 'propped', 'prosper'],
+      ['proper', 'propped', 'cropper', 'dropper', 'popper', 'prosper'],
       'should suggest alternatives'
     )
 
     st.deepEqual(
       us.suggest('Ghandi'),
-      ['shandy', 'Brandi', 'Ghana', 'Grand', 'Grandee', 'Grands', 'handy'],
+      [
+        'Brandi',
+        'Ghana',
+        'Grandee',
+        'Hand',
+        'Hands',
+        'Handy',
+        'Hanoi',
+        'Hindi',
+        'Randi',
+        'Shanxi'
+      ],
       'should suggest alternatives'
```

Comment:
The old results favored "shandy" because of two edits of "shandi" and "Ghandy", which we lose because there's no group with "s" and "g" or with "i" and "y". The old results contained "Grand" and "Grands" because of an edit of "Grandi", which we lose because there's no group with "r" and "h".

The suggestions of 'Hand', 'Hands', 'Handy', 'Hanoi', 'Hindi', 'Randi' all come from the truncation of "Ghandi" to "Handi" at index 63 of `firstValues`. It is the only non-edit for consideration in the second call to `generate`. Since the old version coincidentally had a net of 1 more "edit" than the updated version, "Handi" was pushed to index 64 of `firstValues` and thus missed the `size` cutoff.